main.py

```
@@ -0,0 +1,221 @@
+import ell
+from openai import Client
+import os
+import subprocess
```

Ruff

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)